de.dlr.sc.virsat.model.ext.core/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: de.dlr.sc.virsat.model.ext.core;singleton:=true
+Bundle-Version: 4.10.0.qualifier
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName

```

Comment:
Maybe we should hard code the name here rather than using the plugin.properties file.